CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ CMAKE_DEPENDENT_OPTION(
   "Slicer_BUILD_ITKPython" OFF)
 mark_as_superbuild(Slicer_INSTALL_ITKPython)
 
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build Slicer with parameter serializer support" OFF)
```

Comment:
Why is it OFF by default?

This means we won't have jsoncpp in the nightly, and we will need it in the default configuration for some of the things done in the Segmentations module (at least).

Cc: @cpinter